File: src/main/java/fr/yodamad/svn2git/service/util/MigrationConstants.java
package fr.yodamad.svn2git.service.util;

/**
 * Migrations constants
 */
public abstract class MigrationConstants {
    /** Default ref origin for tags. */
    public static final String ORIGIN_TAGS = "origin/tags/";
    /** Default branch. */
    public static final String MASTER = "master";
    /** Git push command. */
    public static final String GIT_PUSH = "git push";
    /** Stars to hide sensitive data. */
    public static final String STARS = "******";
    /** Execution error. */
    public static final int ERROR_CODE = 128;
}

File: src/main/kotlin/fr/yodamad/svn2git/service/util/MigrationConstants.kt
package fr.yodamad.svn2git.service.util

/** Default ref origin for tags.  */
const val ORIGIN_TAGS = "origin/tags/"

/** Stars to hide sensitive data.  */
const val STARS = "******"

/** Execution error.  */
const val ERROR_CODE = 128

/** Command to copy directory on Windows. */
// /J Copy using unbuffered I/O. Recommended for very large files.
const val WIN_COPY_DIR = "Xcopy /E /I /H /Q"

/** Command to copy directory on none Windows. */
// cp -a /source/. /dest/ ("-a" is recursive "." means files and folders including hidden)
const val COPY_DIR = "cp -a"
